codes/write_dirs_files_for_gamess_qcxms.py

```diff
@@ -277,13 +277,13 @@ def write_gamess_input(multiplicity: int, mol: object, molname: str, mol_input_p
             write_gamess_input(multiplicity, mol, molname, mol_input_path)
 
         spectrum_input_path = spectra_dir / ("qcxms" + ".in")
-        qcxms_template = load_template("templates", "qcxms_input_template.in")
+        qcxms_template = load_template("qcxms_input_template.in")
         if not Path(spectrum_input_path).exists():
-            qcxms_params = read_parameters(args.params_filename, ["QC_Program", "QC_Level", "ntraj", "tmax", "tinit", "ieeatm"])
+            qcxms_params = read_parameters(args.params_filename, ["QC_Program", "QC_Level", "ntraj", "tmax", "tinit"])
```

Comment:
Why are we not reading the 'ieeatm' anymore?

Reply:
It was a mistake. Fixed [b6d895c](https://github.com/RECETOX/ei_spectra_predictions/pull/43/commits/b6d895c20db06f75e49f5f55bb4c9584f15267b6)